### PR TITLE
Add FOAM back to labs

### PR DESCRIPTION
**Pull request by samccone**

### CODE REVIEW

**Comment by arthurvr** [comment]
merge conflict needs to be resolved, also those first commit have nothing to do with this PR.

**Comment by samccone** [comment]
^ all fixed sorry about that @arthurvr

**Comment by samccone** [comment]
Fixes #1265

File: index.html [hunk 1]
```diff
@@ -269,7 +269,9 @@
 								<li>
 									<a href="examples/aurelia/" data-source="http://aurelia.io/" data-content="Aurelia is a next generation JavaScript client framework that leverages simple conventions to empower your creativity">Aurelia</a>
 								</li>
-
+								<li class="routing">
+									<a href="examples/foam/" data-source="http://foam-framework.github.io/foam/" data-content="FOAM is a deeply MVC Javascript metaprogramming framework being developed by a team at Google. ">FOAM</a>
```
Comment (arthurvr) [inline index.html]: > `e. "`

extra space over there

Comment (samccone) [inline index.html]: so much      space

**Pull request merged**